packages/preview-button/README.md

```diff
@@ -54,10 +63,81 @@ module.exports = ({ env }) => ({
             type: "products",
           },
         },
+        {
+          uid: "api::vac.vac",
+          query: {
+            type: "vac",
+          },
+          dialog: {
+            enabled: true,
+          },
+        },
       ],
     },
   },
 });
 ```
 
 After the plugin integration, you have to build the Strapi dashboard by using `strapi build && strapi develop` or you can use the `strapi develop --watch-admin`
+
+### Preview a content type by navigating to the preview page
+
+By adding the required configuration, you can add a preview button to the content type. But you have to ensure that the frontend application is able to render the content type.
+Example:
+
+```ts
+"preview-button": {
+    enabled: true,
+    config: {
+      domain: env("FRONTEND_PUBLIC_URL"),
+      token: env("PREVIEW_SECRET_TOKEN"),
+      contentTypes: [
+        {
+          uid: "api::product.product",
+          query: {
+            type: "products",
+          },
+        },
+      ],
+   },
+  },
```

Comment:
Misschien kun je er een volledige JSON tree van maken door met `{` te beginnen en met `}` te eindigen. Dan heb je support voor Prettier, en dan is het makkelijker om syntaxfouten te herkennen.